index.js

```diff
@@ -1,18 +1,9 @@
-let elements = document.getElementsByTagName('*');
-
-for (let i = 0; i < elements.length; i++) {
-    let element = elements[i]
-
-    for (let j = 0; j < element.childNodes.length; j++) {
-        let node = element.childNodes[j];
-
-        if (node.nodeType === 3) {
-            let text = node.nodeValue;
-            let replaced = text.replace('69', 'Nice 😎');
-
-            if (replaced !== text) {
-                node.nodeValue = replaced;
-            }
+const elements = document.getElementsByTagName('*');
+const elementsArray = Array.from(elements);
+elementsArray.forEach(element => {
+    element.childNodes.forEach(node => {
+        if (node.nodeType === 3 && node.nodeValue.includes('69')) {
+            element.innerHTML = node.nodeValue = node.nodeValue.replace('69', '69 <-- Now <strong>That\'s</strong> Nice 😎');
```

Comment:
Could we use something like `⇦` instead of `<--`?

Reply:
yes